torchci/lib/track.ts

```diff
@@ -1,4 +1,14 @@
+import { randomBytes } from "crypto";
```

Comment:
Do you need to `yarn add` this one? it's a bit weird to not see any package update with a new dependency, may be it has already been added by something else

Reply:
I think it might be a default package or something